transformers/test_data/vow_file.go

```diff
@@ -51,17 +52,16 @@ var VowFileHeaderSyncLog = core.HeaderSyncLog{
 	Transformed: false,
 }
 
-var VowFileModel = shared.InsertionModel{
-	SchemaName: "maker",
-	TableName:  "vow_file",
-	OrderedColumns: []string{
-		constants.HeaderFK, "what", "data", constants.LogFK,
+var VowFileModel = event.InsertionModel{
+	SchemaName: constants.MakerSchema,
+	TableName:  constants.VowFileLabel,
+	OrderedColumns: []event.ColumnName{
+		constants.HeaderFK, constants.WhatColumn, constants.DataColumn, constants.LogFK,
 	},
-	ColumnValues: shared.ColumnValues{
-		"what":             "wait",
-		"data":             "100000000000000000000000",
-		constants.HeaderFK: VowFileHeaderSyncLog.HeaderID,
-		constants.LogFK:    VowFileHeaderSyncLog.ID,
+	ColumnValues: event.ColumnValues{
+		constants.WhatColumn: "wait",
```

Comment:
I'm actually more inclined to declaring these locally in all transformers, to make them as self-sufficient as possible. What's your take? :thinking:

Reply:
🤔 My thought was to reduce duplication, since in this case we'd end up defining a WhatColumn constant in several converters. I understand the instinct to make the transformers self-sufficient, but I'm not immediately seeing a downside to using the constants package in these, since we're also including the shared package as well. That said, I don't feel super strongly about it either way, so I'm up for changing it back if folks think that's best.

Reply:
I'm fine with either approach, but I think I would lean toward having shared constants for terminology that's reused across contracts in the dss repo. My thinking is that even if we break down the code in this repo further along any number of abstractions, we'd probably keep event transformers targeting dss together - so we might as well take advantage of its naming scheme.